src/assets/main.css

```
@@ -5,182 +5,304 @@
+
   /* ============================================================
      BOTONES
   ============================================================ */
   .btn {
     @apply inline-flex items-center justify-center gap-2 px-4 py-2.5 rounded-xl text-sm font-semibold border border-transparent select-none cursor-pointer transition-all duration-200 active:scale-95 disabled:opacity-50 disabled:cursor-not-allowed disabled:transform-none;
   }
 
-  /* Azul: Guardar, Detalle, Consultar */
   .btn-primary {
     @apply bg-blue-600 text-white hover:bg-blue-700;
     box-shadow: 0 2px 8px rgba(37,99,235,0.22);
```

Stylelint (17.9.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)